src/assets/icons/iwd-flower.svg

```diff
@@ -0,0 +1,15 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 118.63 100.67">
+	<defs>
+		<!-- <style>.cls-1{fill:#fab532;}.cls-2{fill:#dd812a;}.cls-3{fill:#75284c;}.cls-4{fill:#8ed2c8;}.cls-5{fill:#11afb9;}</style> -->
```

Comment:
This doesn't work with our icon flow and will be stripped out. Moved the styles to "fill" attributes on their respective paths below to get color layers working.